lib/livebook/session.ex

```diff
@@ -1268,4 +1302,11 @@ defmodule Livebook.Session do
 
   defp container_ref_for_section(%{parent_id: nil}), do: :main_flow
   defp container_ref_for_section(section), do: section.id
+
+  defp system_memory_usage(state) do
+    memory = %{runtime: nil, system: Utils.fetch_system_memory()}
+    state = %{state | memory_usage: memory}
```

Comment:
```suggestion
    put_in(state.memory_usage.system, Utils.fetch_system_memory())
```

Reply:
Didn't work.

Reply:
Ah, we need to assign it back to `state`, or did you run into an error?

Reply:
Just ignore that comment, I was doing the changes sequentially. Done!